rediscluster/client.py

```diff
@@ -671,13 +679,17 @@ def _execute_command(self, *args, **kwargs):
                         count=self.connection_pool.nodes.reinitialize_steps,
                     )
             except TimeoutError:
-                log.exception("TimeoutError")
+                logException("TimeoutError")
```

Comment:
Retrying successfully after a timeout affects the user, because they had to wait for the timeout. So I think this should always be an error log.

Reply:
Let's see what they say. After all we were monkey patching the logger to avoid these being logged as errors. So it seems strange to continue needing to monkey patch.